XMonad/Hooks/DynamicBars.hs

```diff
@@ -64,6 +68,9 @@ import qualified XMonad.Util.ExtensibleState as XS
 -- is called when the number of screens changes and on startup.
 --
 
+-- TODO: Make into [(ScreenId, Handle)] instead of keeping the two parallel
+-- lists.
```

Comment:
Are these `TODO` and following `XXX` comment things that should be dealt with now?  Or can they be put off until later?